scss/_patterns_code-copyable.scss

```diff
@@ -3,59 +3,39 @@
 $cc-button-size: 36px;
 
 @mixin vf-p-code-copyable {
-  .p-code-copyable {
-    @extend %bordered-text-vertical-padding;
-    @extend %vf-bg--x-light;
-    @extend %vf-has-round-corners;
-    @extend %vf-is-bordered;
-
-    display: flex;
-    margin-bottom: $input-margin-bottom;
-    overflow: hidden;
-    position: relative;
+  @include deprecate('3.0.0', 'p-code-copyable is being removed, use the code snippet block instead.') {
+    .p-code-copyable {
+      @extend %leading-linux-prompt-icon;
+
+      background-color: $color-code-background;
+      margin-bottom: $input-margin-bottom;
+      overflow: hidden;
+      padding: $spv-inner--x-small--scaleable $sph-inner;
+      position: relative;
 
-    & + & {
-      margin-top: 0; // overrides p + p
+      & + & {
```

Comment:
Is this doing anything? (I know it was here before). It seems to reset top margin, which doesn't seem to be set anywhere. And the code comment itself is confusing as well (mentions `p` as if it was about paragraphs)

Reply:
I couldn't see that it was, but I wasn't confident that removing it wouldn't affect something elsewhere - it was introduced here (funnily, back when "code copyable" was being introduced, after originally being "code snippet"): https://github.com/canonical-web-and-design/vanilla-framework/commit/e29bdfecc8452fcd442470a9ae99d4e3180d0d0f#diff-4cce33a2f0a480da043e242e6c3213fe2f0ca05816fc29719f8500b6d3c65771R16

Reply:
OK. Doesn't hurt to keep it I guess, until we remove it in next major version.